XLS-0065d - Single Asset Vault (#128)
ripple/opensource.ripple.com: docs/xls-65d-single-asset-vault/concepts/pseudo-account.md

A pseudo-account is a special type of account that holds assets on behalf of an on-chain protocol, and is used in the following use cases:

- **Automated Market Makers (AMM)**: The [XLS-30 amendment](https://xrpl.org/resources/known-amendments#amm) introduced pseudo-accounts for AMMs by adding the `AMMID` field to the `AccountRoot` ledger entry. This field links a pseudo-account to an AMM instance, allowing it to track XRP and token balances in the pool and issue `LPTokens` on behalf of the AMM instance.

- **Single Asset Vaults**: A single asset vault pseudo-account is used to store deposited funds and issue MPT shares. A new `VaultID` field is introduced in the `AccountRoot` ledger entry, which links the pseudo-account with the vault.

- **Lending Protocol**: While still in development, this protocol is expected to use pseudo-accounts to hold loan funds.

> A pseudo-account has strict limitations. It cannot receive payments from other accounts, cannot send transactions, as it has no signing authority, and exists solely to store or issue assets, such as vault shares.

Reviewer: ```suggestion
A pseudo-account has strict limitations. It cannot receive payments from other accounts, cannot send transactions since it has no signing authority, and exists solely to store or issue assets.
```
Nitpick. Just felt like a lot of comma pauses.
